pkg/products/threescale/envoyconfig.go

```diff
@@ -123,14 +175,55 @@ var tsHTTPRateLimitFilter = hcm.HttpFilter{
 	- &tsHTTPRateLimitFilter
 	- name: envoy.router
 **/
-func getAPICastHTTPFilters() []*hcm.HttpFilter {
-	httpFilters := []*hcm.HttpFilter{
-		&tsHTTPRateLimitFilter,
-		{
-			Name: "envoy.router",
-		},
+func getAPICastHTTPFilters(installation *integreatlyv1alpha1.RHMI) ([]*hcm.HttpFilter, error) {
```

Comment:
I would break this function into two and remove the requirement of the installation CR for the function

Reply:
Great idea, thank you!